tests/test_helpers/helpers.ts

```diff
@@ -642,7 +643,7 @@ export function getExportedExcelData(model: Model): ExcelWorkbookData {
       handler.exportForExcel(data);
     }
   }
-  return data;
+  return fixLengthySheetNames(data);
```

Comment:
not super nice to do that here, but since it's called at the first line of `getXLSX` it's really hard to test otherwise ...